test/verify/check-packagekit

```diff
@@ -1006,7 +1006,7 @@ ExecStart=/usr/local/bin/{packageName}
         # reboots automatically
         b.switch_to_top()
         b.wait_in_text(".curtains-ct h1", "Disconnected")
-        m.wait_reboot()
+        self.wait_reboot()
         self.allow_restart_journal_messages()
```

Comment:
Quick review, the whole idea of adding `self.wait_reboot` is to call `allow_restart_journal_messages()` automatically so this PR should clean that up.

Reply:
Yes, so here I should remove `self.allow_restart_journal_messages()` because it is being called in `self.wait_reboot()`
@jelly